samples/ConsoleApp/Program.cs

```diff
@@ -1,46 +1,47 @@
 ﻿// Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-using System;
-using System.Collections.Generic;
-using System.Threading;
-using Microsoft.DurableTask;
-using Microsoft.DurableTask.Grpc;
-using Microsoft.Extensions.Logging;
-
-ILoggerFactory loggerFactory = LoggerFactory.Create(builder =>
-{
-    builder.SetMinimumLevel(LogLevel.Debug);
-    builder.AddSimpleConsole(options =>
-    {
-        options.SingleLine = true;
-        options.UseUtcTimestamp = true;
-        options.TimestampFormat = "yyyy-mm-ddThh:mm:ss.ffffffZ ";
-    });
-});
-
-DurableTaskGrpcWorker worker = DurableTaskGrpcWorker.CreateBuilder()
-    .AddTasks(tasks =>
+using Microsoft.DurableTask.Client;
+using Microsoft.DurableTask.Worker;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+IHost host = Host.CreateDefaultBuilder(args)
+    .ConfigureServices(services =>
     {
-        tasks.AddOrchestrator("HelloSequence", async context =>
+        services.AddDurableTaskClient(builder =>
```

Comment:
Right now, client and worker must be registered independently. I am not against worker implementations registering their corresponding client automatically. But I'd like to gather feedback on that before making that choice for the gRPC worker.

Reply:
I feel like the common case would be registering both, and that registering one or the other would be the advanced case. Could we simply have an `AddDurableTask` extension method that registers both? Or is that complicated because we've put them in two separate packages with no top-level package to give developers everything?

Reply:
I agree the common case will probably be registering both _for worker scenarios_ only. I think it will also be common to want to register only the client (e.g., Web API with client, but worker is in a separate console app). I think in the future we should explore `AddDurableTaskWorker` also registering the corresponding client, but yes, the package separation complicates it. I am open to adding a Worker --> Client package reference, but it will require some design around integrating the calls together. I think this can be left to post-GA.